### DISCUSSION TITLE
New Releases template issue

### DISCUSSION THREAD

**Discussion by XmnwildX12** [Chimildic/goofy#232] [Помощь с ошибкой]
I am using the New Releases template and it was working well, I recently added more followed artists and now the script is failing with the following errors:

[TypeError: Cannot read properties of null (reading 'album')]
TypeError: Cannot read properties of null (reading 'artists')

Any ideas on how to get this working again?

**Comment by Chimildic** [comment]
Can you add full text with error in order to see numbers of lines code?

Also save you artists to cache and attach this file from Google Drive
```js
function backupArtists() {
  let artists = Source.getArtists({ followed_include: true })
  Cache.write('backup_artists', artists)
}
```

**Reply by XmnwildX12** [reply]
Here you go

2:06:55 PM	Notice	Execution started
2:08:27 PM	Error	[TypeError: Cannot read properties of null (reading 'album')] '\n\nОписание и решение ошибок: https://chimildic.github.io/goofy/#/errors'
2:08:26 PM	Error	
TypeError: Cannot read properties of null (reading 'artists')
getArtistIds	@ library.gs:1217
(anonymous)	@ library.gs:1205
removeArtists	@ library.gs:1205
(anonymous)	@ library.gs:433
getReleasesByArtists	@ library.gs:430
collectWeekReleases	@ main.gs:22
[backup_artists.zip](https://github.com/Chimildic/goofy/files/10189250/backup_artists.zip)
